icons/apache/apache-line-wordmark.svg

```diff
@@ -1 +1 @@
279C71.9921 55.0301 71.672 55.0409 71.5465 55.0519ZM110.155 59.0086L110.148 62.9934L104.905 63.0005L99.6609 63.0076V61.7568C99.6609 61.0689 99.6525 59.2852 99.6422 57.7931L99.6235 55.0802H98.7553H97.8872V63.9797V72.8793L97.9504 72.9131C97.996 72.9376 98.2362 72.9456 98.8144 72.9417C99.2548 72.9389 99.6179 72.9336 99.6213 72.93C99.6246 72.9265 99.6381 71.0511 99.6512 68.7625L99.675 64.6014H104.912H110.148L110.155 68.7836L110.162 72.9659H111.049H111.936V63.9948V55.0238H111.049H110.162L110.155 59.0086ZM115.896 55.0425C115.886 55.0529 115.877 59.0899 115.877 64.0137V72.9659H121.86H127.843V72.1337V71.3015H122.761H117.679L117.686 67.9514L117.693 64.6014L122.036 64.5942L126.379 64.5871V63.7974V63.0076L122.036 63.0005L117.693 62.9934L117.686 59.855L117.679 56.7166L122.655 56.7095L127.632 56.7023V55.8701V55.0379L121.773 55.0308C118.551 55.0269 115.907 55.0321 115.896 55.0425ZM59.0071 56.744C60.5076 56.7761 60.5243 56.779 61.0563 57.1004C61.3427 57.2734 61.5835 57.4724 61.8948 57.7932C62.2718 58.1818 62.495 58.5145 62.6981 58.9908C62.9379 59.5534 63.005 59.9176 63.0048 60.6578C63.0046 61.4869 62.9325 61.9406 62.722 62.4373C62.1663 63.7487 61.2052 64.5421 60.0115 64.6749C59.8263 64.6956 58.7121 64.7116 56.8603 64.7203L53.9957 64.7337V60.7297C53.9957 58.5276 54.001 56.7206 54.0074 56.7141C54.0275 56.6939 57.6922 56.7158 59.0071 56.744ZM42.2744 57.4288C42.2917 57.4947 42.4291 57.847 42.5795 58.2116C42.73 58.5762 42.8821 58.9571 42.9176 59.0579C42.9531 59.1588 43.1024 59.5206 43.2494 59.8619C43.3965 60.2033 43.6137 60.7342 43.7322 61.0418C43.8507 61.3493 44.0432 61.838 44.1601 62.1279C44.277 62.4177 44.5158 63.0103 44.6908 63.4447C44.8657 63.8792 45.0417 64.3091 45.082 64.4C45.1222 64.491 45.1729 64.6243 45.1945 64.6962C45.2162 64.7682 45.3231 65.0302 45.4322 65.2784C45.5413 65.5267 45.6386 65.7632 45.6486 65.804C45.6652 65.8725 45.6582 65.8797 45.5589 65.897C45.4209 65.9208 38.5957 65.9533 38.5957 65.9301C38.5957 65.9205 38.8285 65.351 39.1131 64.6646C39.8838 62.8054 40.5387 61.2243 41.0029 60.1017C41.2307 59.5509 41.544 58.7956 41.6992 58.4232C41.8544 58.0508 42.0124 57.6636 42.0505 57.5628C42.1611 57.2697 42.2228 57.2327 42.2744 57.4288ZM72.1366 57.5557C72.2337 57.7494 72.4378 58.2465 72.8287 59.2413C73.0177 59.7223 73.1982 60.1793 73.2299 60.2569C73.2615 60.3345 73.3436 60.5376 73.4123 60.7083C73.7366 61.5137 74.0944 62.3833 74.2597 62.7677C74.3597 63.0004 74.4702 63.2733 74.5051 63.3742C74.54 63.4751 74.6142 63.6655 74.6699 63.7974C74.7255 63.9292 74.8467 64.2339 74.9391 64.4744C75.0315 64.7149 75.1976 65.1319 75.3081 65.401C75.4187 65.6701 75.5011 65.8983 75.4914 65.9081C75.4648 65.9348 70.0973 65.9513 69.2088 65.9274C68.7852 65.9161 68.4385 65.8961 68.4385 65.883C68.4385 65.86 68.7009 65.2261 69.9734 62.1752C70.31 61.3684 70.6268 60.6067 70.6775 60.4826C70.8487 60.0638 71.2179 59.1888 71.4929 58.5501C71.6433 58.201 71.7982 57.8265 71.8372 57.7179C71.9328 57.4517 71.9508 57.4217 72.0145 57.4217C72.0518 57.4217 72.091 57.4648 72.1366 57.5557Z"/></g></svg>
```

Comment:
this still has multiple `path` elements, but I guess since they are grouped, it can be considered united?

Reply:
Added another commit that merges them in one path, it seems to work correctly on icomoon.